ui/.env.test

```diff
@@ -1,5 +1,29 @@
-COOKIE_SECRET="keyboardmash"
-MAGIC_LINK_SECRET="potatopotatopotato"
-DEPLOY_URL="localhost:3001"
-LANDING_PAGE_URL="https://cobudget.webflow.io"
-FROM_EMAIL="noreply@cobudget.com"
+COOKIE_SECRET="A_LONG_RANDOM_VALUE_THAT_IS_NOT_THIS_STRING"
+DATABASE_URL="postgres://postgres:postgres@localhost:35432/cobudget"
+POSTMARK_FROM_EMAIL="fake"
+POSTMARK_API_TOKEN="fake"
+MAGIC_LINK_SECRET="83uj*U8j3r!@3"
+FROM_EMAIL="test@test.com"
+FACEBOOK_APP_ID="656966848728529"
+FACEBOOK_APP_SECRET="9406673d6841ded46f6d269d920f4d14"
```

Comment:
These are testing vars so doesn't hurt much to have public but since we're not gonna be logging into fb in our tests we also don't need it here